esp-hal-common/src/soc/esp32/radio_clocks.rs

```diff
@@ -1,3 +1,17 @@
+//! Radio clocks driver(ESP32)
```

Comment:
```suggestion
//! Radio clocks driver (ESP32)
```

Reply:
Every similar mistake has been fixed, thank you!